### ISSUE TITLE
Applying arbitration fee

### ISSUE THREAD

**Issue by refund-agent2** [bisq-network/proposals#335]
I've been gradually been not reimbursing the full amount sent to DAO address at delayed payouts since Cycle 22, but there was no clear documentation about it or proper discussion, as @pazza83 [pointed out](https://github.com/bisq-network/roles/issues/93#issuecomment-835922865).
This arbitration fee is kept by the DAO and should at least cover DAO's arbitration expenses, but also could generate a revenue to cover regular costs and serve as an insurance when something goes [wrong](https://github.com/bisq-network/support/issues/822) in the refund process.
It will be applied for all cases where a trader is not responding. It will never be applied for bugs when involved parties are doing their best to help to fix it.
The arbitration fee will be calculated by substracting a % of the security deposit from the protocol violator, as in the next table:
|Trade amount|Arbitration fee|
|------------|----------------|
|<0.01       |30.00%          |
|0.01-0.1    |20.00%          |
|0.1-2       |10.00%           |

In example, for a 0.012 BTC trade with a 15% security deposit where the seller is not responding, where the delayed payout would be a total of 0.0156 BTC, the buyer would receive 0.01533BTC and DAO would keep 0.00027BTC.

The above table is a guide for reference, and not a mandatory rule. It's based on the maximum amounts I have been imposing since Cycle 22. The refund agent is just a middleman between traders and the DAO, which is the final judge of every trade since the deposit tx is sent to the donation address.

**Comment by leo816** [comment]
Thank you @pazza83 and @refund-agent2 . Will add it to the bisq.wiki

**Comment by pazza83** [comment]
Hi @refund-agent2 

Thanks for the write up.

I think the costs of arbitration should be covered by the trade fees. No trader ever really knows when you might need it. 

I like the idea of DAO insurance for when things go wrong, but in reality the BTC being burned is not differentiated in anyway. 

With regards the fees suggested as a percentage of security deposit I think a standard percentage of security deposit should be used.

I worry that the proposed changes will mean that trades that have not been successful in mediation due to an unresponsive trader will end up with less BTC than proposed by the mediator.

Using your example:

>In example, for a 0.012 BTC trade with a 15% security deposit where the seller is not responding, where the delayed payout would be a total of 0.0156 BTC, the buyer would receive 0.01533BTC and DAO would keep 0.00027BTC.

Trade amount: 0.015 BTC
Buyer security deposit: 0.0018
Seller security deposit: 0.0018


**Mediator proposal**

If seller is not responding mediator will propose.

Buyer to receive: 0.00181
Seller: to receive: 0.0005

**Refund agent proposed payout**

If seller is still not responding refund agent will propose.

Buyer to receive: 0.01533

**Result**

Buyer would be 0.00277 BTC worse off though no fault of their own an inconvenienced more by the process going to arbitration.

I do not think the party not at fault should be in a worse position as a result of their counter party being unresponsive.

I think it would be good to establish if there is a consensus for users entering arbitration. And if so in what cases.

I will vote this down as I do not think the current proposal is a fair outcome for all occasions.

**Comment by refund-agent2** [comment]
You are correct, arbitration should never be worse than mediation for the part who did the right, unless mediator did a mistake.
What I have been already doing in this cases is just giving the winner the amount decided by the mediator and leave the rest for the DAO. Buyer would be refunded 0.00181 BTC and the 0.0005 BTC that should be awarded to seller if accepted mediation is now kept for the DAO. It's not buyer's fault that the other part is not responding, but neither is Bisq's.

> I think the costs of arbitration should be covered by the trade fees. No trader ever really knows when you might need it.

Traders who need arbitration don't pay the arbitration fee, which is paid by those who aren't using it by not responding. More than 80% of arbitration cases at last cycle were caused by unresponsive traders. That has a remarkable cost for Bisq and increases the risk that Burningman role supposes for the DAO.

>I like the idea of DAO insurance for when things go wrong, but in reality the BTC being burned is not differentiated in anyway.

It would still be better than fixing mistakes just by printing more BSQ when DAO's budget has a shortfall. A not detailed  insurance budget is better than no budget at all, although this budget could be estimated from my reports if necessary.

>With regards the fees suggested as a percentage of security deposit I think a standard percentage of security deposit should be used.

I don't think mediators use this method to calculate their payouts. I have to manually calculate this payouts, so I have chosen an easy way.
The percentages used in the table are the max amounts and the main goal of arbitration is still to reimburse and compensate affected traders. I will surely reduce the arbitration fee if a trader used a 50% security deposit.

**Comment by sqrrm** [comment]
I think this is a good refund agent scheme. The option is for the trader to wait for a refund request to the DAO which would take much longer and also run the risk of not being accepted. This way the trader gets refunded quickly and in full while there is less pressure put on the DAO.

**Comment by pazza83** [comment]
Thanks @refund-agent2 for answering my concerns.

> Traders who need arbitration don't pay the arbitration fee, which is paid by those who aren't using it by not responding. More than 80% of arbitration cases at last cycle were caused by unresponsive traders. That has a remarkable cost for Bisq and increases the risk that Burningman role supposes for the DAO.

Indirectly the traders entering arbitration would pay for it by receiving a lessor amount in compensation. I still think the costs of arbitration should be covered by trade fees in the same way everything else is. The extra amount could then be allocated to an insurance amount in the same way you propose above.

Also I think that if going down the route of having unresponsive traders pay arbitration fees, it does not make sense not to expand this out to other users making unsuccessful arbitration cases.

- Traders trading with incorrect bank details
- Traders not making payment in time
- Traders paying partial amounts 
- Traders using BTC as reference 

Basically anyone that has been penalized by the mediator and then does not accept mediation, and then subsequently fails to prove their case to the arbitrator.

> The arbitration fee will be calculated by substracting a % of the security deposit from the protocol violator, as in the next table:

Also the way the arbitration fee is proposed means that users choosing to trade with larger security deposits would be impacted more than those choosing to trade with smaller security deposits. I would suggest a percentage of the trade amount would be fairer.  But still not as fair as including arbitration costs in trade fees.

**Comment by refund-agent2** [comment]
> Also the way the arbitration fee is proposed means that users choosing to trade with larger security deposits would be impacted more than those choosing to trade with smaller security deposits. I would suggest a percentage of the trade amount would be fairer. But still not as fair as including arbitration costs in trade fees.

I found that [mediation penalties](https://bisq.wiki/Table_of_penalties) are based on a trade % amount which benefits protocol violators who used a small security deposit: It doesn't matter if the penalty is 20% of the trade amount when the security deposit is 15% of the trade amount.
Both methods have pros and cons, I think using the same method would make it easier for mediators and refund agents.

**Comment by pazza83** [comment]
Following discussion with @refund-agent2 on Keybase 

I would be in agreement with the following:

## Arbitration Fees

An arbitration fee will be applied in all cases where a trader is not responding. 

An arbitration fee will never be applied for bugs when involved parties are doing their best to help to fix it.

Effectively, the arbitration fee is applied by not reimbursing the full amount sent to the DAO address. Unless mediator has made a mistake, the arbitration fee should not make the disadvantaged trader to receive a lower reimbursement at mediation than at arbitration.

The arbitration fee will be calculated by substracting a % of the security deposit from the protocol violator, as in the next table:

|Trade amount|Arbitration fee|
|------------|----------------|
|<0.01       |30.00%          |
|0.01-0.1    |20.00%          |
|0.1-2       |10.00%           |

The table above sets the maximum amount of arbitration fee based on a security deposit %.

**Comment by pazza83** [comment]
Closing as approved as this is now happening and there have been no additional views to the contrary
